task16.js

```diff
@@ -0,0 +1,17 @@
+// Write a function that checks if a string is a palindrome.
+
```

Comment:
ok for both implementation, but: 
isPalindrome("Madam"); should return `true`. Now it is as `false`